src/OpenTelemetry.AutoInstrumentation.Native/environment_variables.h

```diff
@@ -34,9 +34,15 @@ const WSTRING include_process_names = WStr("OTEL_DOTNET_AUTO_INCLUDE_PROCESSES")
 // "MyApp.exe,dotnet.exe"
 const WSTRING exclude_process_names = WStr("OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES");
 
-// Sets a list of integrations to disable. All other integrations will remain
-// enabled. If not set (default), all integrations are enabled. Supports
-// multiple values separated with comma, for example:
+// Sets a list of integrations to enable. If not set (default), all integrations are enabled.
+// Supports multiple values separated with comma, for example:
+// "ElasticsearchNet,AspNetWebApi2"
+const WSTRING enabled_integrations =
```

Comment:
add this here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/6c20a738c2ffd92ba5a3e374d0c571552d3c8bd5/src/OpenTelemetry.AutoInstrumentation.Native/otel_profiler_constants.h#L12